applications/benchmark/ShapeFunctionMemoization.cpp

```diff
@@ -109,8 +109,8 @@ struct Lobatto
     }
 };
 
-auto testFunction = NuTo::ShapeFunctions3D::DerivativeShapeFunctionsBrickSpectralOrder4;
-using result = Eigen::Matrix<double, 125, 3>;
+auto testFunction = NuTo::ShapeFunctions3D::DerivativeShapeFunctionsTetrahedronOrder2;
+using result = Eigen::MatrixXd;
```

Comment:
Now, `result` can just be replaced by `Eigen::MatrixXd` at the appropriate places. This just causes confusion imho.